docs/tutorial/5_downloading_data.md

```diff
@@ -61,71 +104,94 @@ To do this we can slice the dataset so that we're only accessing the portion of
 ds
 ```
 
-```{code-cell} ipython3
-:tags: [skip-execution]
+```{code-cell} ipython
+ds[0, 0].files
+```
 
+```{code-cell} ipython
+---
+tags: [skip-execution]
+---
 ds[0, 0].files.download()
 ```
 
-Since the `download()` method set up the transfer through globus, you can also check on the status of your transfer in the web app at [app.globus.org](https://app.globus.org).
+```{code-cell} ipython
+ds[0, 0].plot()
+```
+
+The default download directory used by `donwload()` is in the same folder as the ASDF file we loaded, so in this case `~/sunpy/data/VISP/BKPLX`.
+Since the `download()` method set up the transfer through globus, you can check on the status of your download in the activity tab of the web app as we saw earlier.
 
 We can change the download location of the files using the `path` argument.
 But remember that whatever path you specify must be accessible by Globus Connect Personal.
 
-```{code-cell} ipython3
-:tags: [skip-execution]
-
-ds[0, 0].files.download(path="~/not/really/a/path/")  # will hang for a while and then fail
+```{code-cell} ipython
+---
+tags: [skip-execution]
+---
+ds[0].files.download(path="~/somewhere/globus/cant/reach/")  # will hang for a while and then fail
 ```
 
 The `path` keyword will replace placeholders in the path in the same way as we saw with `Fido.fetch()`.
+Any key in the dataset inventory (`ds.meta['inventory']`) can be used for this.
 So for example:
 
-```{code-cell} ipython3
-:tags: [skip-execution]
-
-ds[0, 0].files.download(path="~/dkist_data/{instrument}_{dataset_id}")
+```{code-cell} ipython
+---
+tags: [skip-execution]
+---
+ds[0, 0].files.download(path="~/sunpy/data/{dataset_id}")
 ```
 
-would save the file to `~/dkist_data/BKPLX/VISP_2023_10_16T18_21_47_508_00630200_I_BKPLX_L1.fits`.
+would save the file to `~/sunpy/data/BKPLX/VISP_2023_10_16T18_21_47_508_00630200_I_BKPLX_L1.fits`.
```

Comment:
I think the old path was correct?

Reply:
Apparently not: https://github.com/DKISTDC/dkist/pull/493/files#diff-3ff51c45d1596c1bd89d0cb3d25e72af6e140c93e33faf7f76e0b6024384f7c9R178